internal/agent/testdata/device.yaml

```diff
@@ -18,7 +18,7 @@ spec:
             - contents:
                 source: >-
                   data:,This%20system%20is%20managed%20by%20flightctl.%0A
-              mode: 600
+              mode: 0600
```

Comment:
A small error that I fixed on the way it's actually not relevant to this change. I will remove